react-frontend/src/components/Resources/fordesigners.js

```diff
@@ -16,10 +16,10 @@ function ForDesigners() {
     <ContentBlockContainer id="forDesigners">
       <Row>
         <Col sm={12}>
-          <Row>
+          <Row style={{ marginLeft: '0px', marginRight: '16px' }}>
```

Comment:
This should be a styled component, not repeated on every Row, I believe the file it would go in would be the 'html' styled component file